packages/fxa-payments-server/src/utils.js

```diff
@@ -0,0 +1,74 @@
+import { ActionType as PromiseActionType } from 'redux-promise-middleware';
+const { assign } = Object;
+
+export const parseParams = params => params
+  .substr(1)
+  .split('&')
+  .reduce((acc, curr) => {
+    const parts = curr.split('=').map(decodeURIComponent);
+    acc[parts[0]] = parts[1];
+    return acc;
+  }, {});
+
+export const mapToObject = (list, mapFn) => {
+  const out = {};
+  for (let item of list) {
+    out[item] = mapFn(item);
+  }
+  return out;
+};
+
+export const apiGet = (...args) => apiFetch('GET', ...args);
+
+export const apiDelete = (...args) => apiFetch('DELETE', ...args);
+
+export const apiPost = (accessToken, path, body) =>
+  apiFetch('POST', accessToken, path, { body: JSON.stringify(body) });
+
+export const apiFetch = (method, accessToken, path, options = {}) => {
```

Comment:
From what we discussed, I don't think we need retries here, right? Though I'm not clear what will happen when one of these fails.

Reply:
I don't think we were talking about retries in the client as much as in the server APIs between microservices. Here, we'll get an exception thrown and should have UX to report the error